onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.h

```diff
@@ -40,5 +43,10 @@ class NnapiExecutionProvider : public IExecutionProvider {
   const std::unordered_set<std::string> partitioning_stop_ops_;
 
   std::unordered_map<std::string, std::unique_ptr<onnxruntime::nnapi::Model>> nnapi_models_;
+
+  // nnapi handle for either Android NNAPI or x86 hooker.
```

Comment:
not quite clear to me what "x86 hooker" means
```suggestion
  // nnapi handle for either Android NNAPI or stub implementation.
```